workspace/teampipeline/generate_report.py

```diff
@@ -0,0 +1,203 @@
+import argparse
+import json
+import os
+
+import requests
+
+
+URL = "https://api.github.com/graphql"
+TOKEN = os.environ["DATA_TEAM_GITHUB_API_TOKEN"]  # requires "read:project" and "repo"
+HEADERS = {
+    "Content-Type": "application/json",
+    "Accept": "application/json",
+    "Authorization": f"Bearer {TOKEN}",
+    "GraphQL-Features": "projects_next_graphql",
+}
+ORG_NAME = "opensafely-core"
+
+
+def post_request(payload):  # pragma: no cover
+    rsp = requests.post(URL, headers=HEADERS, json=payload)
```

Comment:
`URL`, `TOKEN`, and `HEADERS` are only used here so I'm not sure there's any value in having them abstracted away as module-level vars.